core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -663,8 +663,6 @@ class KafkaController(val config: KafkaConfig,
    *
    * @param topicPartition The reassigning partition
    * @param reassignment The new reassignment
-   *
-   * @return The updated assignment state
    */
   private def updateCurrentReassignment(topicPartition: TopicPartition, reassignment: ReplicaAssignment): Unit = {
     val currentAssignment = controllerContext.partitionFullReplicaAssignment(topicPartition)
```

Comment:
nit: for completed, could you also add `U3. ....` around the comments on lines 678-680